### PR TITLE
Refactor cypress tests for legibility

### PR DESCRIPTION
**Pull request by Badlapje**
Refactored general wayf behaviour test
Refactored consent test

Will refactor the final ones for the mouse/keyboard in the wayf once that functionality is added

### CODE REVIEW

File: theme/cypress/integration/skeune/wayf/WayfGeneralBehaviour.js [hunk 1]
```diff
@@ -3,23 +3,7 @@
...
-        .should('have.length', 5)
-        .eq(2)
-        .should('have.text', 'Login with Connected IdP 3 en');
-
-      // Check if the search field is present
-      cy.get('.wayf__search').should('exist');
-
-      // Check if the eduId is present
-      cy.contains('.remainingIdps__eduId', 'eduID is available as an alternative');
+    // cy.visit('https://engine.vm.openconext.org/functional-testing/wayf');
```
Comment (MKodde) [inline theme/cypress/integration/skeune/wayf/WayfGeneralBehaviour.js:6]: Can be removed too?

**Review by MKodde — APPROVED** [review]
This greatly increases readability of the cypress tests!

I changed the merge target to the ui-redesign branch? Might not be correct. I guess this PR was based on #929 

Please adapt as you see fit.

File: theme/cypress/integration/skeune/wayf/WayfGeneralBehaviour.js (hunk 1 above)
Comment (Badlapje) [reply theme/cypress/integration/skeune/wayf/WayfGeneralBehaviour.js]: Rebased it for legibility.

Removed this :-)

**Comment by MKodde** [comment]
Make it so:

**Pull request merged**